effcee/cursor.h

```diff
@@ -79,7 +79,7 @@ inline std::string LineMessage(StringPiece text, StringPiece subtext,
   StringPiece full_line = c.RestOfLine();
   const auto* subtext_end = subtext.data() + subtext.size();
   while (subtext_end - (full_line.data() + full_line.size()) > 0) {
-    c.AdvanceLine();
+    c.AdvanceLine  ();
```

Comment:
not a typo?

Reply:
oh, this is a typo.  I want to trigger failure on check-format (which it has!). Yay.

https://btx.cloud.google.com/invocations/6222278d-3e74-4597-a4a4-ef217cc08f18/targets/graphics_shader_compiler%2Feffcee%2Fcheck-format%2Fpresubmit/log